lib/msal-angular/src/msal.guard.ts

```diff
@@ -48,13 +50,18 @@ export class MsalGuard implements CanActivate {
 
     private loginInteractively(url: string): Observable<boolean> {
         if (this.msalGuardConfig.interactionType === InteractionType.Popup) {
+            this.authService.logger.verbose("Guard - logging in by popup");
```

Comment:
Unsure if we want to specify in the logs how it's logging in (popup or redirect) or leave it to be more generic, since the logs from msal-browser are related to events, which just state that login has started, without specifying interaction type

Reply:
I think indicating the interaction type is a good idea.